server/hap2/hatohol/hap2_nagios_ndoutils.py

```diff
@@ -117,6 +118,17 @@ def __parse_url(self, url):
 
         return server, port, database
 
+    def __convert_to_nagios_time(self, hatohol_time):
+        over_second= hatohol_time.split(".")[0]
+        time_array = [char for char in over_second]
+        time_array.insert(4, "-")
+        time_array.insert(7, "-")
+        time_array.insert(10, " ")
+        time_array.insert(13, ":")
+        time_array.insert(16, ":")
+
```

Comment:
One thing is to use a datetime object converted from hatohol_time.

Other is to concatenate partial strings of hatohol time such as
x = hatohol_time[0:4] + "-" + hatohol_time[4:6] + "-".
(However, raw numbers are ugly)